service/geopmdpy/runtime.py

```diff
@@ -85,10 +85,17 @@ def __init__(self, period, num_period=None):
                             units of seconds.
 
             num_period (int): Number of time periods spanned by the
-                              loop.
+                              loop + 1.
```

Comment:
We should clarify in the documentation.  The total loop time is num_periods * period, but since there is no delay in the first loop, there will bee num_period + 1 loop iterations.